portfolio/src/main/webapp/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Hannah's Portfolio</h1>
     </div>
     <div id="comment-container">
       <h3>Feel free to leave comments!</h3>
-      <form action="/data" class="comment-form" method="POST">
+      <form action="/data" id="comment-form" method="POST">
```

Comment:
What's the difference between |class| and |id| here?

Reply:
class can be for multiple objects/things, while id is only for one -- and in this case I'm only having one comment form, so I thought it'd be more accurate to do "id" instead of "class"